{"timestamp":"2025-07-20T21:51:13.471322","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-20T21:51:13.471744","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/weather_etl_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-20T21:51:13.506034Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-20T21:51:13.506695Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-20T21:51:13.507025Z","level":"info","event":"Current task name:extract_weather_data","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-20T21:51:13.507380Z","level":"info","event":"Dag name:weather_etl_pipeline","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-20T21:51:13.988097Z","level":"error","event":"Traceback (most recent call last):","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-20T21:51:13.988670Z","level":"error","event":"  File \"/opt/airflow/scripts/extract_weather.py\", line 113, in <module>","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-20T21:51:13.991225Z","level":"error","event":"    extract_and_load(city_list_path, country=country, limit=record_limit)","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-20T21:51:13.991566Z","level":"error","event":"TypeError: extract_and_load() got an unexpected keyword argument 'country'","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-20T21:51:14.011313Z","level":"info","event":"Error running extract_weather.py: Command '['/home/airflow/.local/bin/python', '/opt/airflow/scripts/extract_weather.py', '/opt/airflow/scripts/city.list.json.gz', 'EG', '1000']' returned non-zero exit status 1.","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-20T21:51:14.011333","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"CalledProcessError","exc_value":"Command '['/home/airflow/.local/bin/python', '/opt/airflow/scripts/extract_weather.py', '/opt/airflow/scripts/city.list.json.gz', 'EG', '1000']' returned non-zero exit status 1.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":877,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1164,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":217,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":240,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/weather_etl_dag.py","lineno":29,"name":"run_extract_script"},{"filename":"/usr/local/lib/python3.12/subprocess.py","lineno":571,"name":"run"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-07-20T21:51:14.051329Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-20T21:51:14.051860Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-20T21:51:14.052130Z","level":"info","event":"Task:<Task(PythonOperator): extract_weather_data>","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-20T21:51:14.052414Z","level":"info","event":"Failure caused by Command '['/home/airflow/.local/bin/python', '/opt/airflow/scripts/extract_weather.py', '/opt/airflow/scripts/city.list.json.gz', 'EG', '1000']' returned non-zero exit status 1.","chan":"stdout","logger":"task"}
